velox/expression/tests/ExpressionFuzzerTest.cpp

```diff
@@ -61,6 +61,7 @@ int main(int argc, char** argv) {
       "in",
       "element_at",
       "width_bucket",
+      "concat",
```

Comment:
Would you add a comment that references the GitHub issue that explains why this function is skipped?